package/npm/packages/elm-format-linux-x64/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@avh4/elm-format-linux-x64",
+  "version": "0.8.6-0",
+  "description": "The Linux 64-bit binary for elm-format.",
+  "repository": "https://github.com/avh4/elm-format",
+  "license": "BSD-3-Clause",
+  "os": [ "linux" ],
+  "cpu": [ "x64" ]
```

Comment:
Do you happen to know where the list of possible cpu values is?  (Or for the short-term, do you know what this value will be for linux arm64 and for MacOS arm64?)

Reply:
The package.json docs weirdly don’t mention what the values are: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#cpu

But this is the list: https://nodejs.org/api/process.html#processarch

> 'arm', 'arm64', 'ia32', 'mips','mipsel', 'ppc', 'ppc64', 's390', 's390x', and 'x64'